### ISSUE TITLE
Install Vim

### ISSUE THREAD

**Issue by ksclarke** [UCLALibrary/packer-samvera#59]
A couple devs use this so let's put it in the build

**Comment by mark-dce** [comment]
Yes please - this should be pretty trivial - we do it like this
https://github.com/curationexperts/ansible-samvera/blob/dd001933a135d0a3d27d291ae13788ced4db92f3/roles/packages/tasks/main.yml#L33
i.e. add `vim` to the package list

**Comment by ksclarke** [comment]
Yes, we have a role for the developer's box now - https://github.com/UCLALibrary/uclalib_role_developers_box - where this should go. We can copy the task syntax Mark links above.
